src/utilities/fe_mesh.jl

```diff
@@ -0,0 +1,162 @@
+# Typ definitions
+type FEBoundary # Tänkte typ parametrisera detta på nåt sätt så att man har FEboundary i 2D, 1D, och 0D
+    dim::Int
+    topology::Array{Int}
+end
+
+type FEMesh{dim, T <: Real}
+    coords::Vector{Vec{dim,T}} # Node coordinates
+    topology::Matrix{Int} # Element nodes
+    boundary::Vector{Vector{}} # Boundary nodes
+end
+
+
+function FEMesh(dim)
```

Comment:
But these new calculated nodes should be stored somehow at least, so we don't have to calculate the coordinates all the time. (in each iteration or whatever). I guess that if you add a quadratic field they will be calculated and stored.